drivers/hmis/spec/models/hmis/form/form_processor_spec.rb

```diff
@@ -2004,6 +2004,19 @@ def process_assessment(new_hud_values: nil)
       expect(cls.information_date).to eq(information_date)
     end
 
+    it 'should work when verified by project ID is provided' do
+      hud_values = {
+        'CurrentLivingSituation.informationDate' => information_date.strftime('%Y-%m-%d'),
+        'CurrentLivingSituation.currentLivingSituation' => 'SAFE_HAVEN', # 118
+        'CurrentLivingSituation.verifiedByProjectId' => p1.id,
```

Comment:
Might be worth adding a test case for re-processing an existing CLS record that already has `verifiedBy` and `verifiedByProjectId` set, and ensuring that verified by changes. (Use-case would be changing the "verified by" value on an existing CLS record that was created in HMIS). For that you can use the normal factory to create the CLS record.